### DISCUSSION TITLE
berkeley-schema: `instrument_name`

### DISCUSSION THREAD

**Discussion by mslarae13** [microbiomedata/nmdc-schema#2132] [General]
@naglepuff presented a local instance of the NMDC data portal using the berkeley-schema. @turbomam pointed out that using `instrument_name` for filtering isn't the best
- it's a string an inconsistent
- we have consistent enumerated values in vendor and model

For some data types, @SamuelPurvine & @corilo, have pointed out, it's VERY important to know what exact instrument generated what data.
So while the model and vendor may both be  `model: solarix_15T`   `vendor: bruker`, the instrument bay be name: 15T FT-ICR MS-1 and name: 15T FT-ICR MS-2.

CURRENTLY in NMDC, the model, vendor, and name are 1:1:1, there are no repeated models and vendors with names.
EVENTUALLY there will be.

Proposal:
Now
- Update NMDC data portal to show `instrument_model` for berk roll out
- Update enum permissible values to have descriptions and title (like we did for https://microbiomedata.github.io/berkeley-schema-fy24/DoiProviderEnum/)

Later
- Post berk, and as a separate squad, re-evaluate the search function associated with instrument
- Add ACTUAL instrument names / identifiers to know when data came from different instruments of the same kind.

Would like opinions on the NOW proposal this week.
@SamuelPurvine @kheal @lamccue @brynnz22

**Comment by kheal** [comment]
@mslarae13  I think we landed on `instrument_model` rather than `instrument_vendor` as the UI-for-now solution per our discussion earlier.  If that's what you meant, then I approve of the NOW proposed actions.

**Reply by kheal** [reply]
Another note of clarification, there are a few instances of multiple models per vendor, but do not think there are multiple vendors per any specific model.  The name:instrument_model are (I believe) 1:1 as you note.

**Comment by kheal** [comment]
Closing this - We've restructured the instrument metadata as documented in this issue: https://github.com/microbiomedata/nmdc-schema/issues/2394, and the instrument names do reflect the models and are interpretable.
